applications/crossbar/priv/couchdb/schemas/tasks.records.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://json-schema.org/draft-03/schema#",
+    "_id": "tasks.records",
+    "additionalProperties": false,
```

Comment:
@fenollp please address @jamesaimonetti  comment.
`clients like to store custom data along side Kazoo data`